deploy/kubernetes/releases/csi-digitalocean-latest.yaml

```diff
@@ -19,6 +19,7 @@
 # example usage: kubectl create -f <this_file>
 #
 
+# This file is only for development use. Do not use in production.
```

Comment:
It might be confusing to users if they find this line in released manifests. Should we omit it or `sed` it out as well during `bump-version`?

Reply:
Ah, yes, sorry, I didn't commit that. Will do.